appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -153,4 +161,16 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
             }
         ];
     }
+
+    private unshiftRuntimeQuickPickItem(runtime: string, runtimeItems: IAzureQuickPickItem<ILinuxRuntimeStack>[]): IAzureQuickPickItem<ILinuxRuntimeStack>[] {
+        const runtimeQuickPickItemIndex: number = runtimeItems.findIndex((qpItem: IAzureQuickPickItem<ILinuxRuntimeStack>) => {
+            return qpItem.data.name === runtime;
+        });
+        if (runtimeQuickPickItemIndex >= 0) {
+            const runtimeQuickPickItem: IAzureQuickPickItem<ILinuxRuntimeStack> = runtimeItems.splice(runtimeQuickPickItemIndex)[0];
+            runtimeItems.unshift(runtimeQuickPickItem);
```

Comment:
So I don't think we want to shift a single runtime. I think we want to move _all_ python runtimes to the top or _all_ node runtimes to the top if it's node. (I realize there's only one python runtime at the moment, but I expect there to be more in the future)

I would use `sort` rather than `unshift` in that case